grc/core/blocks/_build.py

```diff
@@ -50,7 +52,7 @@ def build(id, label='', category='', flags='', documentation='',
 
     cls.inputs_data = _build_ports(inputs, 'sink') if inputs else []
     cls.outputs_data = _build_ports(outputs, 'source') if outputs else []
-    cls.parameters_data = _build_params(parameters or [],
+    cls.parameters_data = _build_params(block_id, parameters or [],
```

Comment:
I suggest to move this to the end of the params

Reply:
@skoslowski I'm afraid "params" has several meanings here. Do you mean moving `block_id` to the end of the function parameters? Or moving `block_id` to the end of the function parameter called `parameters`?

Reply:
I meant, to change to order of parameters on `_build_params`. A parameter that is only used if something goes wrong in log message, should not go first (but rather last) =)

Reply:
That's what I thought :) Fixed!